src/Sulu/Bundle/AdminBundle/Tests/Unit/Controller/AdminControllerTest.php

```diff
@@ -295,14 +295,14 @@ public function testMetadataAction()
         $this->user->getLocale()->willReturn('en');
 
         $metadataProvider = $this->prophesize(MetadataProviderInterface::class);
-        $metadataProvider->getMetadata('pages', 'en')->willReturn($form);
+        $metadataProvider->getMetadata('pages', 'en', [])->willReturn($form);
```

Comment:
Shouldn't there be also a test where some data is passed instead of just an empty array?